File: selector/src/main/java/com/luck/picture/lib/interfaces/OnCallbackListener.java
package com.luck.picture.lib.interfaces;

/**
 * @author：luck
 * @date：2020/4/24 11:48 AM
 * @describe：OnCallbackListener
 */
public interface OnCallbackListener<T> {
    /**
     * @param data
     */
    void onCall(T data);
}

File: selector/src/main/java/com/luck/picture/lib/interfaces/OnCallbackListener.kt
package com.luck.picture.lib.interfaces

/**
 * @author：luck
 * @date：2020/4/24 11:48 AM
 * @describe：OnCallbackListener
 */
interface OnCallbackListener<T> {
    /**
     * @param data
     */
    fun onCall(data: T)
}